external-api/api-tests/v1/party-db.api-test.ts

```diff
@@ -19,12 +19,69 @@ import { api } from '../api';
 const { APIM_MDM_URL } = process.env;
 const { VALID, VALID_WITH_LETTERS } = MOCK_COMPANY_REGISTRATION_NUMBERS;
 const { get } = api(app);
+let axiosMock: MockAdapter;
+
+beforeEach(() => {
+  axiosMock = new MockAdapter(axios);
+
+  axiosMock.onGet(`${APIM_MDM_URL}customers?companyReg=${VALID}`).reply(HttpStatusCode.Ok, {});
+  axiosMock.onGet(`${APIM_MDM_URL}customers?companyReg=${VALID_WITH_LETTERS}`).reply(HttpStatusCode.Ok, {});
+  axiosMock.onGet(`${APIM_MDM_URL}customers/salesforce?companyRegistrationNumber=${VALID}`).reply(HttpStatusCode.Ok, {});
+  axiosMock.onGet(`${APIM_MDM_URL}customers/salesforce?companyRegistrationNumber=${VALID_WITH_LETTERS}`).reply(HttpStatusCode.Ok, {});
+});
+
+afterEach(() => {
+  axiosMock.resetHistory();
+});
+
+describe('when AUTOMATIC_SF_CUSTOMER_CREATION_ENABLED is not true', () => {
```

Comment:
Can we update throughout to be more aligned with other feature flags tests e.g

```ts
describe('when TFM Facility end date feature flag is enabled', () => { ... }
describe('when TFM Facility end date feature flag is disabled', () => { ... }
```

I do like the use of the env var here, much better IMO, but better to be consistent